gitnamehere/point-of-sale-group-project
# Create Payment Screen: Part 2 (cont.)

**Ticket:** [ATS-5: Create Payment Screen: Part 2](https://cs375-pos-group-project.atlassian.net/browse/ATS-5?atlOrigin=eyJpIjoiYjRjODk0NDVhNTEyNGMyZDg4ZDM2ZDEyM2ZjNmVmMzAiLCJwIjoiaiJ9)

- Displayed tips on front-end
- Order calculations update according to applied tip percentage
- Added discount, tip, and total order data to orders table
- Created API endpoint to update order data for discount, type, and total in orders table
- Looks a little ugly, may refactor if time allows

Testing:
- `npm run setup-demo`
- `npm start`
- Create an order at `http://localhost:3000/pos/ordering`
- Go to `http://localhost:3000/pos/payment/?id=1`
- Check if math calculations are correct
- Tax is set to 0.08
- Discount codes are "jotchua" (99% discount) and "michael" (50%)
- Go to `http://localhost:3000/api/orders` to check order data updated
